src/js/events.js

```diff
@@ -52,9 +52,11 @@ var fixEvent = function(event) {
     }
 
     // Handle which other element the event is related to
-    event.relatedTarget = event.fromElement === event.target ?
-      event.toElement :
-      event.fromElement;
+    if(!event.relatedTarget) {
+      event.relatedTarget = event.fromElement === event.target ?
+        event.toElement :
+        event.fromElement;
```

Comment:
I did see the comments about preferring plain ol if/else over ternary. Alas it is quite the beauty </sarcasm>.. I'll attempt a facelift..